src/main/java/ua/softserveinc/tc/controller/admin/AdminMappingController.java

```diff
@@ -41,4 +53,11 @@ public String getAdminCreateAbonnement(Model model) {
         return AdminConstants.ViewNames.CREATE_ABONNEMENT_VIEW;
     }
 
+    @GetMapping(AdminModel.DISCOUNTS)
+    public String getAdminDiscount(Principal principal, Model model){
+        //List<AbonnementDto> abonnements = abonnementsService.findAllAbonements();
```

Comment:
I've removed model from the view and forgottento remove this. 
I'll change it with next commit